src/app/main/component/auth/components/restore-password/restore-password.component.html

```diff
@@ -20,8 +20,11 @@ <h2>{{ 'user.auth.forgot-password.fill-form' | translate }}</h2>
     <div class="validation-email-error" *ngIf="emailField.invalid && emailField.touched">
       <app-error [formElement]="emailField" [emailFieldValue]="emailFieldValue" [controlName]="'email'"></app-error>
     </div>
-    <div class="validation-email-error" *ngIf="emailErrorMessageBackEnd != null">
-      {{ emailErrorMessageBackEnd }}
+    <div class="validation-email-error" *ngIf="emailErrorMessageBackEnd === 'already-sent'">
+      {{ 'user.auth.forgot-password.already-sent' | translate }} : {{ emailFieldValue }}
+    </div>
+    <div class="validation-email-error" *ngIf="emailErrorMessageBackEnd === 'email-not-exist'">
+      {{ 'user.auth.forgot-password.email-not-exist' | translate }} : {{ emailFieldValue }}
```

Comment:
check for existing error validation component 
if there is no - create one 
so we will not duplicate the code

Reply:
I created a new component.